unit/av_test.c

```diff
@@ -226,7 +226,8 @@ av_create_addr_sockaddr_in(char *first_address, int index, void *addr)
 	}
 
 	hints.ai_family = AF_INET;
-	ret = getaddrinfo(first_address, NULL, &hints, &ai);
+	/* port doesn't matter, set port to discard port */
+	ret = getaddrinfo(first_address, "discard", &hints, &ai);
```

Comment:
My only minor concern here is using the service name and not the number.  I personally have very little experience with using the name and I don't know if it's as robust for our usage as simply stuffing `"9"` in there.  But I'm fine with leaving it as coded and then changing it if we hear of any problems.